src/index.html

```diff
@@ -1,13 +1,52 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <div class="card"
+         data-qa="card"
+    >
+
+      <div class="picture-box">
+        <div class="picture-bem1">
+          <img
+            class="picture-bem1__elem"
+            src="images/imac.jpeg"
+            alt="Product's picture"
+        >
+        </div>
+      </div>
+
+      <div class="card-text">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</div>
+      <div class="code-text">Product code: 195434</div>
+
+      <div class="star">
+        <div class="bem-stars">
+          <div class="bem-stars__starrow_1"></div>
+          <div class="bem-stars__starrow_2"></div>
+          <div class="bem-stars__starrow_3"></div>
+          <div class="bem-stars__starrow_4"></div>
+          <div class="bem-stars__starrow_5"></div>
+        </div>
```

Comment:
fix it
https://github.com/mate-academy/layout_product-cards/pull/2400#discussion_r1065028664

Reply:
Not sure to know the way  of how to amended class - modificator or something else, here is not exactly clear